src/components/Slider.js

```diff
@@ -130,7 +131,8 @@ class Slider extends Component {
     } = this.props;
     const selectionWidth = Math.abs(scale(selection[1]) - scale(selection[0]));
     const selectionSorted = Array.from(selection).sort((a, b) => +a - +b);
-    const f = d3Format(handleLabelFormat);
+    const defaultLabelFormatFunction = d3Format(handleLabelFormat);
```

Comment:
can you define this in defaultProps?

Reply:
I actually wanted to do it like this initially, but since we need access to `handleLabelFormat`, which we don't have in the `defaultProps` (since we don't have access to the current props of the component), it wouldn't be possible like this.
We could change the signature to be something like this `formatLabelFunction: (handleLabelFormat, value) => string` but I found it would be quite pointless as `handleLabelFormat` is passed in via props anyways... (it could be that I'm missing something, if that's the case, let me know!)
What would you prefer?

Reply:
works for me, let's ship as is